docs/en/latest/reference/example.md

```diff
@@ -115,15 +115,27 @@ metadata:
   namespace: ingress-apisix
   name: apisix
 spec:
-  controller: apisix.apache.org/apisix-ingress-controller
+  controller: apisix.apache.org/apisix-ingress-controller    # 1
   parameters:
-    apiGroup: apisix.apache.org
-    kind: GatewayProxy
-    name: apisix-config
-    namespace: ingress-apisix
-    scope: Namespace
+    apiGroup: apisix.apache.org    # 2
+    kind: GatewayProxy             # 3
+    name: apisix-config            # 4
+    namespace: ingress-apisix      # 5
+    scope: Namespace               # 6
 ```
 
+❶ The controller name should be customized if you are running multiple distinct instances of the APISIX Ingress Controller in the same cluster (not a single instance with multiple replicas). Each ingress controller instance must use a unique controllerName in its [configuration file](configuration-file.md), and the corresponding GatewayClass should reference that value.
```

Comment:
`and the corresponding GatewayClass should reference that value.`

But the example you provided is IngressClass?

Reply:
Good catch! It's been corrected.